tests/specs/run/net_descriptor/err_iii.out

```diff
@@ -0,0 +1,2 @@
+error: Uncaught (in promise) ConnectionRefused: No connection could be made because the target machine actively refused it. (os error 10061)
+    at async Object.connect (ext:deno_net/01_net.js:587:55)
```

Comment:
I'm a bit confused here. The provided permission and host/port pair matches in the `ipv6.js` file, then why is it erroring?

Reply:
it's refused connection from the host of ipv6 not permission error i don't find any valid ipv6 can make connect with it at some port